database/index.js

```diff
@@ -0,0 +1,25 @@
+const mysql = require('mysql');
+const debug = require('debug')('app:*');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  database: 'booking_service',
+});
+
+connection.connect((err) => {
+  if (err) {
+    debug(`error connecting to db: ${err}`);
+    return;
+  }
+  debug('connected to database...');
+});
+
+const insertMockData = (sql, data) => {
```

Comment:
the declaration of the function suggest that the function can be used not only for inserting, change naming or make the function more specific to match the name.

Reply:
Hey david, thanks for the feedback/helpful review! It's a javascript debugging utility that's meant to work with/extend console.error - I'm just using it for all of the logging atm, but I think you can set up it's environment variables to only log specific debugging messages/focus on parts of your modules. 

Someone in a tutorial I was watching the other day suggested it, and it seems really popular, here's the link to npm - https://www.npmjs.com/package/debug